app/factories/attendee.py

```diff
@@ -11,6 +12,7 @@ class Meta:
         sqlalchemy_session = db.session
 
     event = factory.RelatedFactory(EventFactoryBasic)
+    ticket = factory.RelatedFactory(TicketFactory)
```

Comment:
why?

Reply:
Without doing this, while running tests, some test were failing because of `ticket_id` being `None`

Reply:
Because you made ticket_id nullabse = false, that's why